ui/hooks/useGasFeeInputs.js

```diff
@@ -412,6 +412,9 @@ export function useGasFeeInputs(
       if (networkAndAccountSupports1559) {
         estimatesUnavailableWarning = true;
       }
+      if (gasPriceToUse === '' || gasPriceToUse < 1) {
```

Comment:
Gas price can be lower than 1 on custom networks

Reply:
If gas price can be lower than 1 on custom networks (e.g. 0), how does the code below work for them? 
```
      if (gasPriceToUse === '' || gasPriceToUse === 0) {
        gasErrors.gasPrice = GAS_FORM_ERRORS.GAS_PRICE_TOO_LOW;
      }
```

Reply:
@dan437 I believe he means it can be a decimal? (e.g. 0.1)